### PR TITLE
Resource label selector as a default option for discovered applications

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2283965
A non-blocker BZ, Which is targetted for 4.16 to improve the user experience for GA.

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
image

File: packages/mco/components/discovered-application-wizard/wizard-steps/configuration-step/configuration-step.tsx [hunk 1]
```diff
@@ -66,35 +66,35 @@ export const Configuration: React.FC<ConfigurationProps> = ({
...
+                value={ProtectionMethodType.RECIPE}
+                description={t('Secure namespaces as per recipe definition.')}
+                label={<strong>{t('Recipe')}</strong>}
                 onChange={(event, _unUsed) =>
                   setProtectionMethod(_unUsed, event)
                 }
-                checked={
-                  protectionMethod === ProtectionMethodType.RESOURCE_LABEL
-                }
+                checked={protectionMethod === ProtectionMethodType.RECIPE}
```
Comment (GowthamShanmugam) [inline packages/mco/components/discovered-application-wizard/wizard-steps/configuration-step/configuration-step.tsx:97]: Just interchanging Recipe and Resource label radio component position

File: packages/mco/components/discovered-application-wizard/utils/reducer.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ export const initialState: EnrollDiscoveredApplicationState = {
     name: '',
   },
   configuration: {
-    protectionMethod: ProtectionMethodType.RECIPE,
+    protectionMethod: ProtectionMethodType.RESOURCE_LABEL,
```
Comment (GowthamShanmugam) [inline packages/mco/components/discovered-application-wizard/utils/reducer.ts:69]: Setting Resource label as default option

File: packages/mco/components/discovered-application-wizard/wizard-steps/configuration-step/configuration-step.tsx [hunk 3]
```diff
@@ -66,35 +66,35 @@ export const Configuration: React.FC<ConfigurationProps> = ({
...
-                id="label-based-protection"
+                id="recipe-based-protection"
                 name={RADIO_GROUP_NAME}
-                value={ProtectionMethodType.RESOURCE_LABEL}
-                description={t(
-                  'Secure selected namespace by defining resource label expressions.'
-                )}
-                label={<strong>{t('Resource label')} </strong>}
+                value={ProtectionMethodType.RECIPE}
+                description={t('Secure namespaces as per recipe definition.')}
```
Comment (TimothyAsirJeyasing) [inline packages/mco/components/discovered-application-wizard/wizard-steps/configuration-step/configuration-step.tsx:92]: Secure namespace ?

**Review by TimothyAsirJeyasing — COMMENTED** [review]
One minor comment otherwise LGTM

File: packages/mco/components/discovered-application-wizard/wizard-steps/configuration-step/configuration-step.tsx (hunk 3 above)
Comment (GowthamShanmugam) [reply packages/mco/components/discovered-application-wizard/wizard-steps/configuration-step/configuration-step.tsx:92]: it is for one or more namespaces

Comment (TimothyAsirJeyasing) [reply packages/mco/components/discovered-application-wizard/wizard-steps/configuration-step/configuration-step.tsx:92]: Ack

File: locales/en/plugin__odf-console.json [hunk 4]
```diff
@@ -71,10 +71,10 @@
   "Choose your configuration preference to protect resources (application volumes/PVCs, or Kubernetes objects).": "Choose your configuration preference to protect resources (application volumes/PVCs, or Kubernetes objects).",
   "You have selected {{count}} namespaces, to view or change your selection go back to the previous step._one": "You have selected {{count}} namespaces, to view or change your selection go back to the previous step.",
   "You have selected {{count}} namespaces, to view or change your selection go back to the previous step._other": "You have selected {{count}} namespaces, to view or change your selection go back to the previous step.",
-  "Secure namespaces as per recipe definition.": "Secure namespaces as per recipe definition.",
-  "Recipe": "Recipe",
   "Secure selected namespace by defining resource label expressions.": "Secure selected namespace by defining resource label expressions.",
   "Resource label": "Resource label",
+  "Secure namespaces as per recipe definition.": "Secure namespaces as per recipe definition.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:76]: assuming `Recipe` is a `kind`...
```suggestion
  "Secure namespaces as per Recipe definition.": "Secure namespaces as per recipe definition.",
```

Comment (GowthamShanmugam) [reply locales/en/plugin__odf-console.json]: Changes done

**Review by bipuladh — APPROVED** [review]
/lgtm

**Comment by SanjalKatiyar** [comment]
/hold
fixing unit test

**Pull request merged**